lib/index.lambda.js

```diff
@@ -273,15 +273,3 @@ const generateMessage = (accountCosts, todaysConversionRate) => {
   });
   return message;
 };
-
-if (require.main === module) {
-  process.env.DRY_RUN = process.env.DRY_RUN || "true";
-
-  const event = {};
-  const context = {
-    done: (...args) => {
-      console.log(`context.done(${args.map(arg => String(arg)).join(", ")})`);
-    }
-  };
-  exports.handler(event, context).catch(e => console.error(e));
-}
```

Comment:
This bit moved to `bin/start.js` because it is for running locally, not meant to be deployed